api/scpca_portal/models/project.py

```diff
@@ -254,6 +257,46 @@ def combine_multiplexed_metadata(
 
         return combined_metadata, multiplexed_sample_mapping
 
+    def add_contacts(self, contact_emails, contact_names):
+        """Creates and adds project contacts."""
+        emails = contact_emails.split(";")
```

Comment:
lets pull these out to a constant so that it matches how we handle `\t` and other delimiters